modules/temp-channels/events/interactionCreate.js

```diff
@@ -0,0 +1,108 @@
+    if (interaction.isButton()) {
+        const vc = await client.models['temp-channels']['TempChannel'].findOne({
+            where: {
+                [Op.and]: [
+                    {id: interaction.member.voice.channelId},
+                    {creatorID: interaction.member.id}
+                ]
+            }
+        });
+
+        if (!vc) {
+            interaction.reply({ ephemeral: true,
+                content: interaction.client.configurations['temp-channels']['config']['notInChannel']});
+            return;
+        }
+        if (interaction.customId === 'tempc-add') {
+            const modal = new Modal()
+                .setCustomId('tempc-add-modal')
+                .setTitle(localize('temp-channels', 'add-modal-title'));
+            const userInput = new TextInputComponent()
+                .setCustomId('add-modal-input')
+                .setLabel(localize('temp-channels', 'add-modal-prompt'))
+                .setStyle('SHORT')
+                .setPlaceholder('User#1234');
+            const actionRow = new MessageActionRow().addComponents(userInput);
+            modal.addComponents(actionRow);
+            await interaction.showModal(modal);
+        }
+        if (interaction.customId === 'tempc-remove') {
+            const modal = new Modal()
+                .setCustomId('tempc-remove-modal')
+                .setTitle(localize('temp-channels', 'remove-modal-title'));
+            const userInput = new TextInputComponent()
+                .setCustomId('remove-modal-input')
+                .setLabel(localize('temp-channels', 'remove-modal-prompt'))
+                .setStyle('SHORT')
+                .setPlaceholder('User#1234');
+            const actionRow = new MessageActionRow().addComponents(userInput);
+            modal.addComponents(actionRow);
+            await interaction.showModal(modal);
+        }
+        if (interaction.customId === 'tempc-list') {
+            await interaction.deferReply({ephemeral: true});
+            await usersList(interaction);
+        }
+        if (interaction.customId === 'tempc-private') {
+            await interaction.deferReply({ephemeral: true});
+            await channelMode(interaction, 'buttonPrivate');
+        }
+        if (interaction.customId === 'tempc-public') {
+            await interaction.deferReply({ephemeral: true});
+            await channelMode(interaction, 'buttonPublic');
+        }
+        if (interaction.customId === 'tempc-edit') {
+            const modal = new Modal()
+                .setCustomId('tempc-edit-modal')
+                .setTitle(localize('temp-channels', 'edit-modal-title'));
+            const nsfwInput = new TextInputComponent()
+                .setCustomId('edit-modal-nsfw-input')
+                .setLabel(localize('temp-channels', 'edit-modal-nsfw-prompt'))
+                .setStyle('SHORT')
+                .setPlaceholder(localize('temp-channels', 'edit-modal-nsfw-placeholder'));
```

Comment:
I would suggest using a select-menu in the modal to make selecting easier

Reply:
That's not possible (accoring to ppl on the discord js discord after explaining the problem i just invested 1.5 hours in creating)

Reply:
wtf that's a so basic feature… 
sorry then, my bad :sweat:

Reply:
I also thought it should work. both use an action row, embed takes action row => has to work. but no... it doesn't :hAA: